torch/csrc/jit/codegen/cuda/runtime/helpers.cu

```diff
@@ -27,6 +27,44 @@ __device__ constexpr int64_t ceilDiv(int a, int64_t b) {
   return ceilDiv((int64_t)a, b);
 }
 
+// Monotonic and precise lerp is described here:
+// https://math.stackexchange.com/a/1798323
+__device__ double lerp(double start, double end, double weight) {
+  if (abs(weight) < 0.5) {
```

Comment:
I don't think `abs` is correct here (and I'll need to fix it in eager too), it's needed for complex cases only.

Reply:
Good catch~ Sorry for blindly following eager without double checking the actual implementation :stuck_out_tongue_closed_eyes: